onnx_coreml/_operators_nd.py

```diff
@@ -1320,6 +1322,12 @@ def _convert_pool(builder, node, graph, err):
     if len(node.outputs) == 2:
         return err.unsupported_op_configuration(builder, node, graph, "argmax with pool unsupported")
 
+    if 'ceil_mode' in node.attrs and node.attrs['ceil_mode'] == 1:
```

Comment:
is there any way to achieve the same functionality by adding another layer after pooling?

Reply:
Ceil mode modifies how pooling should compute output shape. So, by adding new layer after pooling will lead to incorrect result
possible work-around
1. Padding before pooling to compensate/simulate desired output shape
2. Allowing floating point stride to adjust the ceil/floor behavior

Reply:
I see. Okay, in future PRs we can explore some of these workarounds.